map.py

```diff
@@ -0,0 +1,36 @@
+#!python3
+""" map currencies by cmc-id """
+
+import json
+import sys
+
+
+def load_json(file_name):
+    """ helper function """
+    with open(file_name) as file:
+        res = json.load(file)
+    return res
+
+
+def curr_map(source, c_map):
+    """ map currencies """
+    return [c_map.get(i, F"ERROR mapping for cmc-id: {i}") for i in source]
+    # for i in source:
+    #     curr_id = c_map.get(i, None)
+    #     # if curr_id is None:
+    #     #     # what to do here ?
+    #     res.append(curr_id)
+    # return res
```

Comment:
Maybe unused commented code should be removed

Reply:
Commented code is removed